src/ChainSafe.Gaming.MetaMask.Unity/MetaMaskSignerExtensions.cs

```diff
@@ -0,0 +1,27 @@
+using ChainSafe.Gaming.Evm.Signers;
+using ChainSafe.Gaming.Web3.Build;
+using ChainSafe.Gaming.Web3.Core;
+
+namespace ChainSafe.Gaming.MetaMask.Unity
+{
+    /// <summary>
+    /// <see cref="MetaMaskSigner"/> extension methods.
+    /// </summary>
+    public static class MetaMaskSignerExtensions
```

Comment:
Personally, I would keep all extension methods related to MetaMask Provider/Signer/TxExec in one class, so that there are fewer files in the project.

Reply:
I think that's a good idea but that's not how other projects in the solution do it.

Reply:
Should I just change this for Metamask?

Reply:
I think it should be kept like this. 
If we want to unify them (LIke walletconnectExtensions for both rovider/signer/txexec go in a single class), we should create an issue for it. But it's a low-prio imo.

Reply:
agreed

Reply:
@kantagara But that's how it's done in every other package besides WalletConnect.

Reply:
I see, I've merged ext methods in both Metamask and WalletConnect